tap_hubspot_beta/streams.py

```diff
@@ -816,6 +816,48 @@ class ContactsAssociationStream(AssociationsV3ParentStream):
         th.Property("id", th.StringType),
     ).to_dict()
 
+class ContactsHistoryPropertiesStream(hubspotHistoryV3Stream):
+    """Contacts History Properties Stream"""
+
+    name = "contacts_history_properties"
+    path = "crm/v3/objects/contacts/{id}"
+    properties_url = "properties/v1/contacts/properties"
+    additional_params = {"propertiesWithHistory": True}
+    parent_stream_type = ContactsV3Stream
+    bulk_child = False
+    records_jsonpath = "$[*]"
+    primary_keys = ["id"]
+
+    base_properties = [
+        th.Property("id", th.StringType),
+        th.Property("createdAt", th.DateTimeType),
+        th.Property("updatedAt", th.DateTimeType),
+        th.Property("archived", th.BooleanType),
+        th.Property("archivedAt", th.DateTimeType),
+        th.Property("propertiesWithHistory", th.CustomType({"type": ["object", "string"]})),
+    ]
+
+    def _write_schema_message(self) -> None:
+        """Write out a SCHEMA message with the stream schema."""
+        for schema_message in self._generate_schema_messages():
+            schema_message.schema = th.PropertiesList(*self.base_properties).to_dict()
+            singer.write_message(schema_message)
+
+class ArchivedStream(hubspotV3Stream):
```

Comment:
Were we supposed to add the `ArchivedStream`?

Reply:
No. Deleting it.